content/en/dora_metrics/_index.md

```diff
@@ -1,29 +1,26 @@
 ---
 title: DORA Metrics
 kind: documentation
-description: Learn how to use DORA metrics to improve and measure software development.
+description: Learn how to use DORA metrics to measure and improve software development.
 aliases:
 - /continuous_integration/dora_metrics
 is_beta: true
 further_reading:
```

Comment:
Under further reading let's add the first link as the sending incident events page, so if they're done with deployments they can go to incidents